src/styles/index.scss

```
@@ -1,3 +1,77 @@
 body {
   margin: 0;
 }
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+}
+
+.center-point {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  height: 5vmin;
+  width: 5vmin;
+  background-color: #f6a603;
+  border-radius: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.minutes-hand {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+  transform: translate(-50%, -100%) rotate(360deg);
+  transform-origin: 50% 100%;
+  z-index: -2;
+  animation: rotate-minute-hand 3600s steps(60) infinite;
+}
+
+.seconds-hand {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  height: 38vmin;
+  width: 1.5vmin;
+  background-color: #2c8000;
+  transform: translate(-50%, -100%) rotate(360deg);
+  transform-origin: 50% 100%;
+  z-index: -1;
+  animation: rotate-second-hand 60s linear infinite;
+}
+
+.stopwatch--speed-up .seconds-hand {
+  animation-duration: 10s;
+}
+
+.stopwatch--speed-up .minutes-hand {
+  animation-duration: 600s;
+}
+
+@keyframes rotate-second-hand {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
+
+@keyframes rotate-minute-hand {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
```

Use only one keyframes
```suggestion
@keyframes rotate-hand {
  from {
    transform: translate(-50%, -100%) rotate(0deg);
  }
  to {
    transform: translate(-50%, -100%) rotate(360deg);
  }
}

```